server/services/spotify-service.js

```diff
@@ -38,11 +38,12 @@ module.exports = class SpotifyService {
     })
   }
 
-  getPlaylists () {
-    // this.setCredentials()
-    return spotifyApi.getMe().then(data => {
-      console.log('getMe() returned', data.body)
-      return spotifyApi.getUserPlaylists(data.body.id, {limit: 50}).then(data => data.body)
-    })
+  async getPlaylists () {
+    const user = await spotifyApi.getMe().then(data => data.body)
+    const sampling = await spotifyApi.getUserPlaylists(user.id, {limit: 1}).then(data => data.body)
+    const pageCount = Math.ceil(sampling.total / limit)
+    const pages = Array.from(Array(pageCount), (_, x) => x)
```

Comment:
Consider `Array.from(Array(pageCount).keys())`. The mapFn argument here tripped me up a bit until I read more.

Reply:
Yeah .keys() is much more readable. Will change that.